src/main/java/uk/gov/hmcts/reform/et/jms/listner/HmcHearingsEventTopicListener.java

```diff
@@ -60,7 +61,7 @@ public void onMessage(JmsBytesMessage message) throws JMSException,
 
                 processHmcMessageService.processEventMessage(hmcMessage);
             }
-        }  catch (JsonProcessingException ex) {
+        }  catch (JsonProcessingException | MessageProcessingException ex) {
```

Comment:
Maybe split this into two catch blocks and modify the message so we can identify which error happened